frontend/src/app/shared/components/storages/file-picker-modal/file-picker-modal.component.ts

```diff
@@ -71,7 +71,16 @@ export class FilePickerModalComponent extends OpModalComponent implements OnInit
     header: this.i18n.t('js.storages.file_links.select'),
     buttons: {
       openStorage: ():string => this.i18n.t('js.storages.open_storage', { storageType: this.locals.storageTypeName as string }),
-      submit: ():string => this.i18n.t('js.storages.file_links.selection_any', { number: this.selectedFileCount }),
+      submit: (count:number):string => {
+        switch (count) {
+          case 0:
+            return this.i18n.t('js.storages.file_links.selection_none');
+          case 1:
+            return this.i18n.t('js.storages.file_links.selection_one');
+          default:
+            return this.i18n.t('js.storages.file_links.selection_any', { number: count });
+        }
```

Comment:
Does this pattern work for all languages? The i18n module itself might have some more indications, I think some language require and extra case for e.g. 2

Reply:
yes, you are right. russian for example has own plural forms for count 2 to 4. Not sure though, how we should solve this now. I checked other parts of the application, and in so many places we have only two options: singular and plural.

any hints?

Reply:
fixed after talking to @oliverguenther